### PR TITLE
[Photos] Update Photos to Xcode 8.2 Beta 1

### PR DESCRIPTION
**Pull request by dalexsoto**
Update Photos to Xcode 8.2 Beta 1

### CODE REVIEW

File: src/photos.cs [hunk 1]
```diff
@@ -1003,6 +1005,14 @@ interface PHPhotoLibrary {
 		[Export ("performChangesAndWait:error:")]
 		bool PerformChangesAndWait (Action changeHandler, out NSError error);
 
+		[iOS (8,2), TV (10,1)]
```
Comment (spouliot) [inline src/photos.cs]: iOS 8.2 ? seems to be confused with Xcode 8.2 ;-) and should be 10.2

File: src/photos.cs [hunk 2]
```diff
@@ -1003,6 +1005,14 @@ interface PHPhotoLibrary {
...
+		[Export ("performCancellableChanges:completionHandler:")]
+		void PerformChanges (PHPhotoLibraryCancellableChangeHandler cancellableChangeHandler, Action<bool, NSError> completionHandler);
+
+		[iOS (8,2), TV (10,1)]
```
Comment (spouliot) [inline src/photos.cs]: same

File: src/photos.cs (hunk 1 above)
Comment (dalexsoto) [inline src/photos.cs]: :facepalm: haha good catch

**Comment by dalexsoto** [comment]
@spouliot fixed, thanks!

**Pull request merged**